.babelrc

```diff
@@ -1,4 +1,4 @@
 {
   "presets": ["movio"],
-  "plugins": []
+  "plugins": ["transform-flow-strip-types"]
```

Comment:
Don't you need `"syntax-flow"` as well, for this to work? https://babeljs.io/docs/plugins/syntax-flow/